CMakeLists.txt

```diff
@@ -9,15 +9,18 @@
 
 cmake_minimum_required( VERSION 3.12 )
 
-find_package( ecbuild 3 REQUIRED HINTS ${CMAKE_CURENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/../ecbuild )
 
 project( iodaconv VERSION 0.0.1 LANGUAGES CXX Fortran )
+list( APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake )
+set( INSTALL_LIB_DIR lib ) # For uniformity across JCSDA packages
```

Comment:
ecbuild takes this as a hint when creating new targets.

Reply:
@rhoneyager-tomorrow are you saying that ecbuild uses `INSTALL_LIB_DIR` as a hint? If so, wouldn't this need to be aware of which platforms use `lib` and which use `lib64`? If this is a valid concern, I think it would be okay to defer solving this to a later PR (ie, coordinated with other JEDI repos).